idx-kotlin/src/main/java/com/okta/idx/kotlin/client/IdxClient.kt

```diff
@@ -291,10 +291,11 @@ private fun Any?.asJsonElement(): JsonElement {
 }
 
 private fun IdxRemediation.Form.Field.selectedOptionToJsonContent(): JsonElement? {
-    if (value == null) {
-        return null
+    val localValue = value ?: return null
+    if (localValue is JsonPrimitive) {
```

Comment:
Is safe cast below not sufficient for your check?

Reply:
Unfortunately, there are 2 different formats, one where it's an object: 

```
{
    "form": {
        "value": [...]
    }
```

And one where it's just a value, say `"foobar"`.

The new check is checking for a primitive, and the next one is checking for the object format.

The added test case failed before this change.